### DISCUSSION TITLE
i need help with coding

### DISCUSSION THREAD

**Discussion by yandelsucksatcoding** [KadeArchive/Kade-Engine#1162] [Q&A]
so in manifest and i click the default json i dont know how to code the data in and i tried to do it and it said invalid object key

**Comment by isakube** [comment]
why does anybody care about manifest filessssssssss
you don't need to modify them when you build from source and i dont think you gotta modify them when not building from sourcee dashkfljahdsfjkladhsfjkalsfhklj

**Reply by yandelsucksatcoding** [reply]
then i just need help with coding then. cause everytime i make a custom song it says UTILIS ERROR CANNOT FIND A TEXT ID ASSET OF ASSETS/DATA/TEST/TEST.JSON all the time i tried getting a newer update and the same result happened. i really need help. i was just told i needed to change the manifest files
